rpmbuild/mock.cfg.j2

```diff
@@ -52,7 +52,7 @@ metadata_expire=0
 cost=1
 best=1
 
-{%- if "priority" in repo and priority is not none %}
+{%- if repo.get("priority") is not none %}
```

Comment:
How it comes that the previous `iss not none` check doesn't work? :-/

Reply:
We checked `priority is not none` instead of `repo["priority"] is not none`. But it must have been wrong, there is no `priority`.

Reply:
Agh... I tested the output of mock config only via frontend's API and [copr-cli](https://github.com/fedora-copr/copr/blob/main/cli/copr_cli/build_config.py#L50) to retrieve `priority` so I totally overlooked this mistake... thanks for noticing it!

BTW I did the same in `mock-custom-build.cfg.j2`. Can you please fix it there too?